src/main/kotlin/nomic/data/dtos/GameDTO.kt

```diff
@@ -19,16 +21,16 @@ import java.time.LocalDate
  * @property title the name of this game
  * @property createDate the date when this game was created
  * @property currentPlayer the id of player whose turn it is
- * @property userId the id of the user that made this game
+ * @property userId the user that made this game
  */
 
 interface GameDTO : Entity<GameDTO> {
     companion object : Entity.Factory<GameDTO>()
     val gameId: Int
     val title: String
     val createDate: LocalDate
-    val currentPlayer: Int
-    val userId: Int
+    val currentPlayer: Int?
+    val user: UserDTO
```

Comment:
Honestly, I wish you had discussed with the team whether modifying DTOs to utilize Ktorm's entity sequences was something that needed to be considered in this user story as this could have saved you a lot of unnecessary work.